vipps-checkout-api.md

```diff
@@ -72,7 +72,13 @@ Once a session is created, it is to be opened inside an iFrame embedded on the m
 
 #### SDK
 
-Vipps provides an SDK to make opening the session on the merchant easy. It is highly recommended to use this.
+Vipps provides an
+[SDK](#step-2-displaying-the-session)
```

Comment:
Just a suggestions, but I'm a bit unsure if we want to direct them straight to step 2? Maybe we should just say something like "we will explain the SDK more in details later on" or something like that?

Reply:
Yes, maybe that's better. Will tweak it a bit.